Makefile

```diff
@@ -112,6 +112,12 @@ deploy-%: | deployment/% deployment/%/secret.json deployment/%/overlay.jsonnet
 show-deploy-%: | deployment/% deployment/%/secret.json deployment/%/overlay.jsonnet  ## Show k8s manifests that would be deployed
 	kubecfg show $(TLA_ARGS) deployment/main.jsonnet
 
+diff-deploy-%: ## Show diff of k8s manifests between files and deployed
```

Comment:
I think this commit should be combined with the one that adds all the other deployment machinery.

Reply:
ok.